docs/reference/hook-command/list-of-hook-commands/state-delete.md

```diff
@@ -3,11 +3,11 @@
 > See also: [state-get](#state-get), [state-set](#state-set)
 
 ## Summary
-Delete server-side-state key value pairs.
+Deletes server-side-state key-value pairs.
```

Comment:
Do we really want the "-"? Not sure what our standard is and if we're consistent across the board?

Reply:
The dash does make sense. But whether to use it or not is often a matter of taste. I'll revert this change as not ready to update this one way or the other across the board.